modules/machine-user-infra-machines-iso.adoc

```diff
@@ -2,19 +2,46 @@
 //
 // * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
 // * post_installation_configuration/node-tasks.adoc
+ifeval::["{context}" == "multi-architecture-configuration"]
+:multi:
+endif::[]
 
 :_content-type: PROCEDURE
 [id="machine-user-infra-machines-iso_{context}"]
-= Creating more {op-system} machines using an ISO image
+= Creating {op-system} machines using an ISO image
 
 You can create more {op-system-first} compute machines for your bare metal cluster by using an ISO image to create the machines.
 
 .Prerequisites
 
 * Obtain the URL of the Ignition config file for the compute machines for your cluster. You uploaded this file to your HTTP server during installation.
+* You installed the OpenShift CLI (`oc`)
```

Comment:
I have seen this prerequisite step written multiple ways, but I think it might help to make it a definitive prereq if you added "must" to it. So something like:
```suggestion
* You must have the OpenShift CLI (`oc`) installed.
```